CONTRIBUTING.md

```diff
@@ -130,11 +133,30 @@ func (ssa *SQLStorageAuthority) AddPerson(p Person) (error) {
 }
 ```
 
-Add a migration with
+You will also need to update the `initTables` function from `sa/database.go` to
+tell Gorp which table to use for your versioned model structs. Make sure to
+consult the flag you defined so that only **one** of the table maps is added at
+any given time, otherwise Gorp will error.  Depending on your table you may also
```

Comment:
It's pretty confusing and there _might_ be a work around, but since I knew it worked correctly with only one mapping at a time I decided to skip the trek into Gorp-guts to figure out why it was breaking.